### DISCUSSION TITLE
Download link

### DISCUSSION THREAD

**Discussion by dlucasjr** [GeoPyTool/GeoPyTool#15] [Q&A]
Hello!

I cannot download the files from the baidu link on geopytool.com, so I wanted to know if there's another link to download the files. 

Thanks!

**Comment by cycleuser** [comment]
In fact, the link there is the only way that I can share files as I am currently staying in somewhere the Google Drive got blocked.

Finally, after some attempts, I am trying to upload one copy of the file to Google Drive at here https://drive.google.com/drive/folders/1vSneUfJII2_agpJc1kDZpMXZtkCjLHpU?usp=sharing

Hope it will be available to you.

**Reply by dlucasjr** [reply]
It's working fine, thank you!

If you want I can upload this (and future updates as well) to my own Drive and I could provide the link for you to share on Download page. If you're interested just let me know!
